app/src/main/java/space/taran/arkmemo/models/Version.kt

```diff
@@ -0,0 +1,19 @@
+package space.taran.arkmemo.models
+
+import android.os.Parcelable
+import kotlinx.parcelize.IgnoredOnParcel
+import kotlinx.parcelize.Parcelize
+import space.taran.arkmemo.data.VersionMeta
+import space.taran.arkmemo.data.repositories.ResourceId
+
+@Parcelize
+data class Version(
+    val content: Content,
```

Comment:
Actually, `title` from `content` is also metadata, it will be moved into `MetadataStorage` in the future.